src/simulation/SimulationManager.java

```diff
@@ -385,8 +395,25 @@ public String getCurrentStateStringForProperty(String property) {
         return statePath.getCurNode().stringForProperty(property);
     }
 
-    public String getCurrentStateDiffString() {
-        return statePath.getCurNode().getDiffString(statePath.getPrevNode());
+    /**
+     * getCurrentStateDiffStringFromLastCommit returns the diff between the current
+     * and the previous last-committed state.
+     * @return String
+     */
+    public String getCurrentStateDiffStringFromLastCommit() {
```

Comment:
What's the purpose of calling this method instead of calling `getCurrentStateDiffStringByDelta(1)`?